.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:8-jdk
+      - image: circleci/openjdk:8u171-jdk
```

Comment:
?

Reply:
this is more advanced jdk version because it fails on the build that this version solve the issue

Reply:
only 8 should bring the latest, you should look up why it's failing, it would be a problem if you have compabilities  issues